src/cache/disk.rs

```diff
@@ -173,7 +173,7 @@ impl Storage for DiskCache {
     ) -> Result<()> {
         let key = normalize_key(key);
         let mut buf = vec![];
-        preprocessor_cache_entry.write(&mut buf)?;
+        preprocessor_cache_entry.serialize_to(&mut buf)?;
```

Comment:
It's weird that you do a mix of amends and additional commits to fixup things.

Reply:
Yeah, sorry, I usually always to amends, but I was starting to take too much time to do history editing. I should probably stick to new commits in this case, unless otherwise asked (like with the time macros thing above).